addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -356,6 +356,14 @@ body.editor_enable.o_basic_theme.o_in_iframe {
                 overflow-y: scroll;
                 max-height: 200px;
             }
+
+            #justify , #style.dropup, #font-size.dropup {
+                > .dropdown-menu {
+                    transform: unset !important;
```

Comment:
A little comment would be welcome here because without knowing the context, this looks like alarming stuff :-P